packages/cardpay-sdk/sdk/utils/signing-utils.ts

```diff
@@ -435,3 +435,15 @@ function sortSignaturesAsBytes(
     return [contractSignature, ownerSignature];
   }
 }
+
+export function signatureToVRS(rawSignature: string) {
```

Comment:
is this essentially the same as `ethSignSignatureToRSVForSafe()` in this module?

Reply:
It looks like it. However, there is one diference, in the recovery identifier (`v`):

`signatureToVRS` uses `parseInt(value)` while `ethSignSignatureToRSVForSafe` uses `parseInt(value, 16)`

So for example, let's have:
`signature = 0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c`

`v` is the last 2 bytes and that is `1c`

`signatureToVRS`:

`parseInt("1c") `
`=> 1`

`ethSignSignatureToRSVForSafe`:

`parseInt("1c", 16)`
`=> 28`

Judging by [this article](https://medium.com/mycrypto/the-magic-of-digital-signatures-on-ethereum-98fe184dc9c7) it looks like the `v` can be either either 27 (0x1b) or 28 (0x1c). That makes me think `ethSignSignatureToRSVForSafe` is the correct implementation and we should use it instead of `signatureToVRS` (and get rid of it). 

However I'm not sure how this even worked then, given the parsing might be wrong. 🙇‍♂️ Judging by this comment https://github.com/ethereum/go-ethereum/issues/19751#issuecomment-504900739, `v` can also be 0 or 1 and this is the reason both implementations that we have are working.